contracts/protocol/libraries/position/LimitOrder.sol

```diff
@@ -40,7 +40,7 @@ library LimitOrder {
         uint120 _remainSize
     ) internal {
         // remainingSize should be negative
-        _self.partialFilled += _self.size - _remainSize;
+        _self.partialFilled += (_self.size - _self.partialFilled - _remainSize);
```

Comment:
this calculation only takes effect for the index that has partialFilled. the other work like old way cause `partialFilled == 0`